dist/obs-server.spec

```diff
@@ -601,7 +601,7 @@ fi
 %postun -n obs-api
 %insserv_cleanup
 %service_del_postun %{obs_api_support_scripts}
-%restart_on_update apache2
+%restart_on_update_reload apache2
```

Comment:
But that is not defined. `_restart_on_update_reload` is.

Reply:
Done.